src/Translator/src/DependencyInjection/UxTranslatorExtension.php

```diff
@@ -43,12 +43,12 @@ public function prepend(ContainerBuilder $container)
         if (!$this->isAssetMapperAvailable($container)) {
             return;
         }
-
+        $config = $container->getExtensionConfig('ux_translator')[0];
         $container->prependExtensionConfig('framework', [
             'asset_mapper' => [
                 'paths' => [
                     __DIR__.'/../../assets/dist' => '@symfony/ux-translator',
-                    '%kernel.project_dir%/var/translations' => 'var/translations',
+                    $config['dump_directory'] => '@app/translations',
```

Comment:
Maybe we should add a check that the dum_directory config exists? Or are we 100% sure that this conf exist

Reply:
And the configuration is defined with a default value so it should always be set